src/Esfa.Roaao.Xslx.IntegrationTests/RoaaoExcelTests.cs

```diff
@@ -73,5 +73,61 @@ public void ShouldNotHaveOverlappingPeriods()
             }
             Assert.IsTrue(errors.Count == 0, string.Join(Environment.NewLine, errors));
         }
+
+        [TestMethod]
+        public void ShouldNotHaveDuplicateCurrentPeriods()
+        {
+            List<string> errors = new List<string>();
```

Comment:
we tend to use weak typing for easier reading:
var errors = new List<string>();